test/metabase/query_processor_test/advanced_math_test.clj

```diff
@@ -63,28 +65,46 @@
                    int)))))
 
 
-(defn- test-aggregation
-  [agg]
-  (->> {:aggregation [agg]}
-       (mt/run-mbql-query venues)
-       mt/rows
-       ffirst
-       double
-       (u/round-to-decimals 2)))
+(defn- aggregation=
+  [expected agg]
+  (testing "As a top-level aggregation"
+    (let [query (mt/mbql-query venues
+                  {:aggregation [agg]})]
+      (mt/with-native-query-testing-context query
+        (is (= expected
+               (ffirst
+                (mt/formatted-rows [1.0]
+                  (mt/process-query query))))))))
+  (when (driver/database-supports? driver/*driver* :expression-aggregations (mt/db))
+    (testing "Inside an expression aggregation"
+      (let [query (mt/mbql-query venues
+                    {:aggregation [[:+ agg 1]]})]
+        (mt/with-native-query-testing-context query
+          (is (= (+ expected 1.0)
+                 (ffirst
+                  (mt/formatted-rows [1.0]
+                    (mt/process-query query))))))))))
+
+;;; there is a test for standard deviation itself
+;;; in [[metabase.query-processor-test.aggregation-test/standard-deviation-test]]
 
 (deftest test-variance
   (mt/test-drivers (mt/normal-drivers-with-feature :standard-deviation-aggregations)
-    (is (= 0.59 (test-aggregation [:var [:field (mt/id :venues :price) nil]])))))
+    (aggregation= 0.6
```

Comment:
Changed the rounding here to 1 decimal place instead of 2 because there were tiny differences in the MongoDB answer